model-engine/model_engine_server/infra/gateways/abs_llm_artifact_gateway.py

```diff
@@ -73,3 +74,11 @@ def get_model_weights_urls(self, owner: str, model_name: str, **kwargs) -> List[
         for blob_name in container_client.list_blob_names(name_starts_with=prefix):
             model_files.append(f"https://{account}.blob.core.windows.net/{bucket}/{blob_name}")
         return model_files
+
+    def get_model_config(self, path: str, **kwargs) -> Dict[str, Any]:
```

Comment:
Are these changes related...? I think these are @squeakymouse ?

Reply:
need to add this function since i updated `LLMArtifactGateway`. not quite sure how to test though